tests/unit_test/twitter_ads_api/twitter_ads_api_pipeline_test.py

```diff
@@ -489,9 +556,11 @@ def test_should_call_func_for_every_seven_days_till_final_end_date(
         get_current_final_end_date_mock: MagicMock,
         get_param_dict_from_api_query_parameters_mock: MagicMock
     ):
-        iter_dict_from_bq_query_for_bigquery_source_config_mock.return_value = (
-            [{'entity_id': 'id_1', 'start_date': '2022-08-01'}]
-        )
+        iter_dict_from_bq_query_for_bigquery_source_config_mock.return_value = ([{
+            'entity_id': 'id_1',
+            'campaign_start_date': CAMPAIGN_START_DATE_1,
+            'start_date': START_DATE_1
```

Comment:
in particular inline date here. maybe use constants in other tests where the date isn't tested or matter so much (but use date constants throughout the test)